vme/src/file_index_type.cpp

```diff
@@ -191,6 +190,11 @@ zone_type *file_index_type::getZone() const
     return m_zone;
 }
 
+std::string file_index_type::getID() const
```

Comment:
We've called this the symbolic name / aka .symname in DIL.

Reply:
Ah - should probably use that then instead